contentcuration/contentcuration/frontend/channelEdit/vuex/clipboard/actions.js

```diff
@@ -382,7 +382,7 @@ export function moveClipboardNodes(context, { legacyTrees, newTrees, target }) {
   }
   if (newTrees.length) {
     for (let copyNode of newTrees) {
-      promises.append(
```

Comment:
I do this more often than I care to admit.... this may be the first time it got committed though...

Reply:
I've done it too